docs/pages/access-controls/device-trust/jamf-integration.mdx

```diff
@@ -29,31 +29,44 @@ and behavior.
 
 (!docs/pages/includes/device-trust/prereqs.mdx!)
 
-## Step 1/4. Create a Jamf user
+## Step 1/4. Create Jamf API credentials
 
-Create a readonly Jamf user for inventory sync.
-
-1. Access `https://yourtenant.jamfcloud.com/accounts.html`, replacing
-   `yourtenant` with your Jamf Pro account.
+<Admonition type="notice" title="v16.0.0 or lower">
+  Teleport versions v16.0.0 or lower don't support Jamf API credentials. Follow
+  the instructions under [Using Jamf user and password authentication](
+  #optional-using-jamf-user-and-password-authentication) instead.
+</Admonition>
 
-1. Create a new Standard Account with the following settings:
+Follow the instructions at [Jamf API Roles and Clients](
+https://learn.jamf.com/en-US/bundle/jamf-pro-documentation-current/page/API_Roles_and_Clients.html)
+to create a role and an API client. We recommend creating a role and API client
+specific for Teleport.
 
-   - Username: teleport (change as desired)
-   - Access Level: Full Access
-   - Privilege Set: Custom
-   - Access Status: Enabled
-   - Password: (a strong password of your choice)
-   - Privileges:
-     - Advanced Computer Searches: Read
-     - Computers: Read
+Make sure that your Jamf role has the "Read Computers" privilege.
 
-   Take note of the user and password created here for the next step.
+You can test your client credentials using the following Jamf query:
 
-   User account setup: ![Jamf user setup](
-   ../../../img/access-controls/device-trust/jamf-setup-1.png)
+```code
+URL='<Var name="(=jamf.api_endpoint=)" description="Jamf API URL"/>'
+CLIENT_ID='<Var name="(=jamf.client_id=)" description="Jamf API client ID" />'
+CLIENT_SECRET='<Var name="(=jamf.client_secret=)" description="Jamf API client secret" />'
+
+## Acquire access token from Jamf.
+TOKEN_RESP="$(curl -X POST "$URL/api/oauth/token" \
+  -H 'Content-Type: application/x-www-form-urlencoded' \
+  --data-urlencode 'grant_type=client_credentials' \
+  --data-urlencode "client_id=$CLIENT_ID" \
+  --data-urlencode "client_secret=$CLIENT_SECRET" -s)"
+echo "$TOKEN_RESP"
+ACCESS_TOKEN="$(echo "$TOKEN_RESP" | jq -r .access_token)"
+
+## Test access token privileges.
+curl -H "Authorization: Bearer $ACCESS_TOKEN" "$URL/api/v1/computers-inventory?page-size=1"
```

Comment:
```suggestion
$ TOKEN_RESP="$(curl -X POST "$URL/api/oauth/token" \
  -H 'Content-Type: application/x-www-form-urlencoded' \
  --data-urlencode 'grant_type=client_credentials' \
  --data-urlencode "client_id=$CLIENT_ID" \
  --data-urlencode "client_secret=$CLIENT_SECRET" -s)"
$ echo "$TOKEN_RESP"
$ ACCESS_TOKEN="$(echo "$TOKEN_RESP" | jq -r .access_token)"

## Test access token privileges.
$ curl -H "Authorization: Bearer $ACCESS_TOKEN" "$URL/api/v1/computers-inventory?page-size=1"
```

Adding prompts so this renders with the right contrast

Reply:
Oh, so that's why it was looking washed.  🤦 

Added to the variable declarations above too.